transition_functions/src/phase0/block_processing.rs

```diff
@@ -485,15 +486,15 @@ mod spec_tests {
     }
 
     // Test files for `process_block_header` are named `block.*` and contain `BeaconBlock`s.
-    processing_tests! {
+    processing_tests!(
         process_block_header,
         |_, state, block: Phase0BeaconBlock<_>, _| unphased::process_block_header(state, &block),
         "block",
         "consensus-spec-tests/tests/mainnet/phase0/operations/block_header/*/*",
-        "consensus-spec-tests/tests/minimal/phase0/operations/block_header/*/*",
-    }
+        "consensus-spec-tests/tests/minimal/phase0/operations/block_header/*/*"
```

Comment:
I see a TON of changes that are either a space removed, a comma removed or a semicolon added. Neither of these changes are related to this pull request.

What is more, some of these changes you made break macros, like this one. Now it produces error:

```
error: unexpected end of macro invocation
   --> transition_functions/src/phase0/block_processing.rs:494:80
    |
432 |     macro_rules! processing_tests {
    |     ----------------------------- when calling this macro
...
494 |         "consensus-spec-tests/tests/minimal/phase0/operations/block_header/*/*"
    |                                                                                ^ missing tokens in macro arguments
    |
note: while trying to match `,`
   --> transition_functions/src/phase0/block_processing.rs:438:34
    |
438 |             $minimal_glob:literal,
```

Reply:
Why did you make these changes?

Reply:
@povi Looks like these were because of running ‘cargo fmt —all’. Let me remove those.

Reply:
@povi Thanks for pointing this out, I reverted all the formatting changes which are not related to this PR. How do I invoke the CI workflow on my latest commit?

Reply:
@shreyas-londhe you should be able to fork the repo and push the changes to branches in your local repo. This should trigger CI.